lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -433,7 +433,7 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
-            <?php if ($show_social_share_link) : ?>
+            <?php if ($show_social_share_link == 1) : ?>
```

Comment:
do you need the `== 1`? I thought `$show_social_share_link` itself would be `true` or `false` so you could just have that?